nninit.lua

```diff
@@ -270,4 +270,95 @@ nninit.sparse = function(module, tensor, sparsity)
   return module
 end
 
+--[[
+--  Aghajanyan, A. (2017)
+--  Convolution Aware Initialization
+--  arXiv preprint arXiv:1702.06295
+--]]
+nninit.convaware = function(module, tensor, options)
+  -- The author of the paper provided a reference implementation for Keras: https://github.com/farizrahman4u/keras-contrib/pull/60
+
+  local sizes = tensor:size()
+  if #sizes ~= 4 then
+    error("nninit.convaware only supports 2d convolutions, feel free to issue a pull request to extend this implementation")
+  end
+
+  -- Make sure that the signal library is available, which provides the Fourier transform
+  local requireSignal = function()
```

Comment:
Would lift the loading of `signal` once to the top of the file (not the function) by using [this concise syntax](https://github.com/Kaixhin/Atari/blob/master/Model.lua#L5), and then throwing an error if it's unavailable right at the top of this function (no need to check the tensor size if `signal` isn't available).